src/app/(proper_react)/(redesign)/(public)/LandingView.tsx

```diff
@@ -44,7 +46,14 @@ export const View = (props: Props) => {
   return (
     <main className={styles.wrapper}>
       <nav className={styles.nav}>
-        <h1>{props.l10n.getString("public-nav-name")}</h1>
+        <VisuallyHidden>
+          <h1>{props.l10n.getString("public-nav-name")}</h1>
+        </VisuallyHidden>
```

Comment:
Isn't this redundant with the alt text from the image? We can just wrap the image in an `<h1>` and be done with it I think.